valohai/internals/global_state_loader.py

```diff
@@ -0,0 +1,227 @@
+import argparse
+import json
+import os
+import sys
+from typing import Any, Dict, List, Set, Tuple, Union
+
+from valohai.internals import global_state
+from valohai.internals.input_info import InputInfo
+from valohai.paths import get_inputs_config_path, get_parameters_config_path
+
+
+def load_global_state(
+    default_inputs_from_prepare: Dict = None,
+    default_parameters_from_prepare: Dict = None,
```

Comment:
Same `Optional` thing here... Maybe the `Dict`s should have some typing info too?